.github/workflows/build.yml

```diff
@@ -135,6 +135,32 @@ jobs:
       - run: yarn test:e2e:ganache:ws:${{ matrix.browser }}
         shell: bash
 
+  black-box:
```

Comment:
It should be invoked on **tag** push and **release branch** only for saving time in normal PRs.